WooCommerce/Classes/ViewRelated/Orders/Order Details/Product List Section/ProductDetailsTableViewCell.swift

```diff
@@ -53,7 +54,7 @@ private extension ProductDetailsTableViewCell {
     }
 
     func configureProductImageView() {
-        productImageView.image = .productPlaceholderImage
+        productImageView.image = UIImage.productPlaceholderImage.imageWithTintColor(UIColor.listIcon)
```

Comment:
I used everywhere `imageWithTintColor` since I don't know if there are some cases where we used another `imageWithTintColor` (in that case, the `tintColor` doesn't work like expected). For this specific case, because we set always the same image, I followed your suggestion ;) 4ff484feb58e9cbefb012d37cf74ce5dafed6319